### PR TITLE
Update several packages for ip@5 transition

### PR DESCRIPTION
**Pull request by AlexanderRichert-NOAA**
## Description

This PR updates the ufs-utils, g2, and grib-util recipes to add the latest versions and allow us to move forward with ip@5.

## Issue(s) addressed

Addresses https://github.com/JCSDA/spack-stack/issues/1020

## Dependencies

--

## Impact

--

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have run the unit tests before creating the PR

### CODE REVIEW

File: var/spack/repos/builtin/packages/grib-util/package.py [hunk 1]
```diff
@@ -30,11 +31,13 @@ class GribUtil(CMakePackage):
     depends_on("w3emc +extradeps", when="@1.2.4:")
     depends_on("w3emc precision=4,d", when="^w3emc@2.10:")
     depends_on("w3nco", when="@:1.2.3")
-    depends_on("g2")
+    depends_on("g2@3.4.9:")
```
Comment (climbfuji) [inline var/spack/repos/builtin/packages/grib-util/package.py]: when="@1.4.0:" ?

Comment (climbfuji) [reply var/spack/repos/builtin/packages/grib-util/package.py]: and similar for earlier versions?

Comment (AlexanderRichert-NOAA) [reply var/spack/repos/builtin/packages/grib-util/package.py]: D'oh, yep. Just fixed for 1.4.0, if it's alright I'll do the other versions in the next batch of NCEPLIBS recipe updates.

**Comment by climbfuji** [comment]
Yes, let's keep the changes to a minimum for the release branch. Have you tested building the unified-env and running the affected applications?

**Comment by AlexanderRichert-NOAA** [comment]
Testing is in progress

**Review by srherbener — APPROVED** [review]

**Pull request merged**